caluma-demo/translations/en-us.yaml

```diff
@@ -0,0 +1,22 @@
+caluma:
+  form-builder:
+    global:
+      optional: optional
+      save: Save
+    question:
+      widgetOverrides:
+        powerselect: Powerselect
+    form:
+      all: All petitions
+      new: New petition 
+      empty: You dont' have any petitions yet.
+      name: Name
+      slug: Slug
+      description: Description
+      isArchived: Archived
+      isPublished: Published
+    notification:
+      form:
+        create:
+          error: Error
```

Comment:
Why are you overriding calumas already existing translations? Just use `en` as locale instead of `en-us`

Reply:
Oops didn't know that was how it worked. Will change it.